test/e2e/flowaggregator_test.go

```diff
@@ -889,6 +1059,64 @@ func getCollectorOutput(t *testing.T, srcIP, dstIP, srcPort string, isDstService
 	return collectorOutput, recordSlices
 }
 
+// getClickHouseOutput queries clickhouse with built-in client and checks if we have
+// received all the expected records for a given flow with source IP, destination IP
+// and source port. We send source port to ignore the control flows during the
+// iperf test.
+func getClickHouseOutput(t *testing.T, data *TestData, srcIP, dstIP, srcPort string, isDstService, checkAllRecords bool) []*ClickHouseFullRow {
+	var flowRecords []*ClickHouseFullRow
+	var queryOutput string
+
+	query := fmt.Sprintf("SELECT * FROM flows WHERE (sourceIP = '%s') AND (destinationIP = '%s')", srcIP, dstIP)
+	if isDstService {
+		query = fmt.Sprintf("SELECT * FROM flows WHERE (sourceIP = '%s') AND (destinationClusterIP = '%s')", srcIP, dstIP)
+	}
+	if len(srcPort) > 0 {
+		query = fmt.Sprintf("%s AND (sourceTransportPort = %s)", query, srcPort)
+	}
+	cmd := []string{
+		"clickhouse-client",
+		"--date_time_output_format=iso",
+		"--format=JSONEachRow",
+		fmt.Sprintf("--query=%s", query),
+	}
+	err := wait.PollImmediate(500*time.Millisecond, aggregatorInactiveFlowRecordTimeout, func() (bool, error) {
```

Comment:
The same question as last time: `aggregatorInactiveFlowRecordTimeout` is 6s but I think it's not related to when the flow record should theoretically arrive on Clickhouse after we start the traffic. I assume `clickHouseCommitInterval ` will be a better variable to represent this timeout?

Reply:
For IPFIX collector the polling timeout is `exporterActiveFlowExportTimeout+aggregatorActiveFlowRecordTimeout*2`. Do you think on top of that by adding 2 times the default commit interval of 1s should be good enough?

Reply:
If the check on the Clickhouse side is always after passing the check on the ipfix collector, 2s sounds good to me here.

Reply:
addressed. Thanks.